src/groups/mqb/mqbs/mqbs_filestore.h

```diff
@@ -220,6 +220,8 @@ class FileStore : public DataStore {
     typedef StorageCollectionUtil::StorageMapIter      StorageMapIter;
     typedef StorageCollectionUtil::StorageMapConstIter StorageMapConstIter;
 
+    typedef bsl::vector<mqbi::Queue*> ReplicationNotificationVec;
```

Comment:
Vector here is better than list because
1. we reserve and reuse its capacity
2. the memory block is continuous